security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersDeserializer.java

```diff
@@ -51,16 +49,17 @@ public SecurityAnalysisParameters deserialize(JsonParser parser, Deserialization
 
                 case "extensions":
                     parser.nextToken();
-                    extensions = JsonUtil.readExtensions(parser, deserializationContext, JsonSecurityAnalysisParameters.getExtensionSerializers());
+                    if (parameters.getExtensions().size() > 0) {
```

Comment:
Same as for load flow parameters : I think it's simpler to call the "update" event when there is no extension.